itest/starboard-operator/behavior/behavior.go

```diff
@@ -79,15 +50,15 @@ func VulnerabilityScannerBehavior(inputs *Inputs) func() {
 
 				err := inputs.Create(ctx, deploy)
 				Expect(err).ToNot(HaveOccurred())
-				Eventually(inputs.HasActiveReplicaSet(inputs.PrimaryNamespace, deploy.Name), inputs.AssertTimeout).Should(BeTrue())
+				Eventually(inputs.HasActiveReplicaSet(inputs.PrimaryNamespace, deploy.Name), 300*time.Second).Should(BeTrue())
```

Comment:
I see we are updating AssertTimeout to 5min, so we dont need this change here. Let it come from inputs.Asserttimeout

Reply:
will address in the next commit.